src/main/java/com/auctionapp/api/configuration/SecurityConfiguration.java

```diff
@@ -64,6 +64,7 @@ public void configure(HttpSecurity http) throws Exception {
                 .permitAll()
                 .antMatchers("/api/auth/**").permitAll()
                 .antMatchers("/api/auctions/**").permitAll()
+                .antMatchers("/api/bids/highestBid/**").permitAll()
```

Comment:
Everyone can access this endpoint?

Reply:
Yes, as it is needed for display of info on the product overview page for the section that even guest users are allowed to see.